### ISSUE TITLE
Meta Q - Docs and Maintainers

### ISSUE THREAD

**Issue by XakV** [vagrant-libvirt/vagrant-libvirt#940]
Hello maintainers, I'm not sure how to ask this, but do you need additional help maintaining the repo? The README seems to be outdated and open issues and pull requests seem to be stagnating. This is a really good tool for development and I am offering to help if needed.

**Comment by infernix** [comment]
We could use help, yes; I've been unable to find time the last 18 months and don't expect to have time the next year orso either. @electrofelix is pulling most of the weight right now, @pradels is MIA for a long time and @pronix is likely also too busy do to much. I'll defer the decision making to @pronix and @electrofelix.

**Comment by XakV** [comment]
Thanks. I'll start a read through in a bit either way. I know you don't simply just show up and help maintain a project.

**Comment by jwcmd** [comment]
Hey @XakV  did you have any trouble setting up the dev environment? I tried both debian and ubuntu16 but the steps in the development section in the readme didn't get me to a state where I could run the tests without error and I haven't had more time to investigate yet. Wondering which invious steps I'm missing.

**Comment by electrofelix** [comment]
Going to close this based on updated docs now published at https://vagrant-libvirt.github.io/vagrant-libvirt/, improved development steps #1623, and #1069 covering the need for additional project maintainers.

@XakV if you do happen to have a bit of time, any help is appreciated, I'm hopeful the docs and tests added over the last couple of years help make it a little easier. And while some aspects might require a bit of upfront discussion, mostly I think if you find something that makes the project a bit easier for you to use, just putting together an initial stab is possibly the best approach as often I discover I've forgotten how certain areas work unless I've worked on them very recently.

@jwcmd hopefully the adjusted development steps are a little simpler to follow now, and a lot of bugs on getting them to pass on various distros have been fixed over the last few years.
